assets/src/block-templates/reality-check/template.js

```diff
@@ -0,0 +1,32 @@
+import metadata from './block.json';
+import mainThemeUrl from '../main-theme-url';
+
+const column = ['core/column', {}, [
+  ['core/group', {}, [
+    ['core/image', {
+      align: 'center',
+      className: 'mb-0 force-no-lightbox force-no-caption',
+      url: `${mainThemeUrl}/images/placeholders/placeholder-75x75.jpg`
+    }],
+    ['core/heading', {
+      style: {typography: {fontSize: '4rem'}},
+      textAlign: 'center',
+      placeholder: 'Enter title'
```

Comment:
These placeholders should be translatable, you can add this functionality by writing it as `__('Enter title', 'planet4-blocks-backend')` (using WordPress' i18n `__` function)

Reply:
Thank you @mleray, just made the changes.